invokeai/frontend/web/src/services/api/endpoints/models.ts

```diff
@@ -248,9 +301,9 @@ export const modelsApi = api.injectEndpoints({
       DeleteMainModelResponse,
       DeleteMainModelArg
     >({
-      query: ({ base_model, model_name }) => {
+      query: ({ base_model, model_name, model_type }) => {
         return {
-          url: `models/${base_model}/main/${model_name}`,
+          url: `models/${base_model}/${model_type}/${model_name}`,
```

Comment:
Is the intention for the `getMainModels` query to also return ONNX models?

Reply:
The change allows you to retrieve any model type with this query, ie `controlnet`, 'vae', 'lora' are all valid `model_type`s. I don't think this should be changed

Reply:
Ahh, I was confused by the git diff viewer. These changes are the delete model mutation, not the get model query.